install/install.sh

```diff
@@ -20,18 +24,18 @@
 #   -B, --base-url
 #     Override the base URL used for downloading releases
 
-set -euo pipefail
```

Comment:
`set -o pipefail`, on shells that _do_ support it, is _very_ useful, though.

Many shells have `test ±o` but you can probably enable this portably by doing:

    set -eu
    (set -o pipefail) >/dev/null 2>&1 && set -o pipefail || :

Reply:
"Probably" doesn't cut it. Need to be sure.